README.md

```diff
@@ -298,3 +298,37 @@ MirrorMaker2 should size `tasks.max` so that each MirrorSourceConnector task has
 * **Why it matters:** The number of records per task shows the aggregate throughput that Kafka Connect is processing. Each MSC task should be balanced with a similar number of overall records, and any MSC task with many more or fewer records indicates an unbalanced MirrorMaker2 replication state. This likely indicates a hot partition, or an incorrect setting for `tasks.max`.
 
 
+### I am getting `InvalidRecordException` exception with timestamp out of range error
```

Comment:
Kind of nitpicky - should this be "Why am I getting `InvalidRecordException: Timestamp ##### of message with offset ## is out of range` exception?"

Reply:
updated